### DISCUSSION TITLE
Moving away from community contributions

### DISCUSSION THREAD

**Discussion by mattpocock** [mattpocock/ts-error-translator#179] [Announcements]
Hey folks,

I've decided to change the TS Error Translator so that it prioritises _requesting_ error translations from me, instead of asking folks to contribute directly.

The most obvious change is now, when you encounter an error that hasn't been seen before, you'll see a link to **request** a new translation instead of **contribute** one.

There are a few reasons for this:

1. It takes much less time for me to write a translation myself than go through multiple review rounds with contributors.
2. I have an extremely high bar for what I consider a good contribution. I want the translations to feel like they're written with the same voice, in the same style. This is too high a cost to pay for most contributors, which I understand.
3. Since releasing TT I am extremely time-pressed, so I'm trying to optimise things where I can.

I hope that, if you've contributed a translation, you don't feel disheartened by this. I'm trying to be as practical as I can while keeping this project maintainable in the limited amount of time I have.

Much love.

**Comment by Soundvessel** [comment]
I think issues of actual bugs might be getting drowned out by all the translation requests. Since June/July several of us are experiencing a stream of errors in VSCode's Extension Host / Extension Host (Remote) output. It might be good to set up issue categories to make sure actual bugs are elevated?

https://github.com/mattpocock/ts-error-translator/issues/376
Also https://github.com/mattpocock/ts-error-translator/issues/373
